packages/code-studio/src/styleguide/Inputs.tsx

```diff
@@ -10,349 +10,356 @@ import {
-              <DateInput />
-              <small className="text-muted">Does not work in mock</small>
-              <br />
-              <h5>Custom Timeselect</h5>
-              <CustomTimeSelect
-                options={this.timeouts}
-                value={customTimeValue}
-                onChange={value => {
-                  this.setState({ customTimeValue: value });
-                }}
-                customText="Custom Timeout"
-                placeholder="Select a timeout"
-                valueToTime={(value: number) => Math.round(value / 1000)}
-                timeToValue={time => time * 1000}
+        <div className="col">
+          <div className="form-group">
+            <h5>Selection Menu</h5>
+            <select defaultValue="0" className="custom-select">
+              <option disabled value="0">
+                Custom Selection
+              </option>
+              <option value="1">One</option>
+              <option value="2">Two</option>
+              <option value="3">Three</option>
+            </select>
+          </div>
+
+          <div className="form-group">
+            <select defaultValue="0" className="custom-select" disabled>
+              <option disabled value="0">
+                Custom Selection
+              </option>
+              <option value="1">One</option>
+              <option value="2">Two</option>
+              <option value="3">Three</option>
+            </select>
+          </div>
+
+          <div className="form-group">
+            <h5>Input with Select</h5>
+            <div className="input-group">
+              <ComboBox
+                options={EXAMPLES}
+                inputPlaceholder="10.128.0.8"
+                searchPlaceholder="Search actions here"
+              />
+            </div>
+            <br />
+            <div className="input-group">
+              <ComboBox
+                options={EXAMPLES}
+                inputPlaceholder="10.128.0.8"
+                searchPlaceholder="Search actions here"
+                disabled
               />
             </div>
           </div>
+
+          <div className="form-group">
+            <h5>AutoCompleteInput</h5>
+            <AutoCompleteInput options={EXAMPLES} noMatchText="No Matches." />
+          </div>
         </div>
 
-        <div className="row">
-          <div className="col">
-            <div className="form-group">
-              <h5>Auto Resize Textarea</h5>
-              <AutoResizeTextarea
-                value={autoResizeTextareaValue}
-                delimiter=" -"
-                onChange={value => {
-                  this.setState({ autoResizeTextareaValue: value });
-                }}
+        <div className="col">
+          <div className="form-group">
+            <h5>Validate Label</h5>
+            <ValidateLabelInput
+              validationError={!on ? 'Toggle is off' : undefined}
+            >
+              <UISwitch on={on} onClick={handleToggleClick} />
```

Comment:
The styling is a bit off:
- without the `labelText` prop this renders as an `inline-block` element when valid, and `inline-block` that is followed by a block `p` for the error message when invalid. Looks misaligned on the styleguide page.
- with the `labelText` there's no space between the label and the toggle switch.

Reply:
We don't use `UISwitch` with `ValidateLabelInput` anywhere at the moment, this could be a separate ticket.